docs/docs/cmd/teams/chat/chat-get.md

```diff
@@ -0,0 +1,52 @@
+# teams chat get
+
+Get a Microsoft Teams chat conversation.
+
+## Usage
+
+```sh
+m365 teams chat get [options]
+```
+
+## Options
+
+`-i, --id [id]`
+: The ID of the chat conversation. Specify either `id`, `name` or `participants`, but not multiple.
+
+`-n, --name [name]`
+: The display name of the chat conversation. Specify either `id`, `name` or `participants`, but not multiple.
+
+`-p, --participants [participants]`
+: A comma-separated list of one or more e-mail addresses. Specify either `id`, `name` or `participants`, but not multiple.
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
```

Comment:
Let's add in remarks, that we're already adding the user currently signed in to CLI as a participant